apps/camera/js/controllers/controls.js

```diff
@@ -57,6 +57,10 @@ ControlsController.prototype.bindEvents = function() {
   this.app.on('timer:cleared', this.onTimerStopped);
   this.app.on('timer:ended', this.onTimerStopped);
 
+  // Settings
+  this.app.on('settings:opened', this.onSettingsOpened);
```

Comment:
@justindarc , unfortunately, set is overridden by the controllers view and filters for data attributes.